tests/test_revision.py

```diff
@@ -1213,3 +1254,239 @@ def test_failure_message(self):
         assert_raises_message(
             RevisionError, "Dependency resolution failed;", list, iter_
         )
+
+
+class InvalidRevisionMapTest(TestBase):
+    def _assert_raises_revision_map(self, map_, except_cls, msg):
+        try:
```

Comment:
**mike bayer** (zzzeek) wrote:

we have a fixture for this:

    def _assert_raises_revision_map(self, map_, except_cls, msg):
        assert_raises_message(
            except_cls, msg, lambda: map_._revision_map
        )

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2379